test/utils/batch.spec.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-disable react/prop-types*/
+
+import React from 'react'
+import { createStore } from 'redux'
+import { Provider as ProviderMock, batch, connect } from '../../src/index.js'
+import * as rtl from 'react-testing-library'
+import 'jest-dom/extend-expect'
+
+describe('React', () => {
+  describe('batch', () => {
+    afterEach(() => rtl.cleanup())
+
+    it('should render only once for batched dispatches', () => {
+      const store = createStore((state, action) => {
+        let newState =
```

Comment:
Please extract the reducer as a separate variable, and use ES6 default arguments to initialize the state, like:

```js
const initialState = {a : 1};
const reducer = (state = initialState, action) => {}
```